static/js/pages/instances/InstanceOverview.tsx

```diff
@@ -68,35 +68,35 @@ const InstanceOverview: FC<Props> = ({ controlTarget, instance, notify }) => {
       <table>
         <tbody>
           <tr>
-            <th>Name</th>
+            <th className="u-text--muted">Name</th>
             <td>{instance.name}</td>
           </tr>
           <tr>
-            <th>Description</th>
+            <th className="u-text--muted">Description</th>
             <td>{instance.description !== "" ? instance.description : "-"}</td>
```

Comment:
As noted in the other review, this seems to work well:
`<td>{instance.description ? instance.description : "-"}</td>`